_data/navbars/reference.yml

```diff
@@ -185,6 +185,19 @@ section:
       path: /reference/architecture/design/l2-interconnect-fabric
     - title: Calico over IP fabrics
       path: /reference/architecture/design/l3-interconnect-fabric
+- title: VPP Dataplane
+  path: /reference/vpp/
+  section:
+  - title: Uplink interface configuration
+    path: /reference/vpp/uplink-configuration
+  - title: Integration architecture
+    path: /reference/vpp/technical-details
+  - title: Host network configuration
+    path: /reference/vpp/host-network
+  - title: Migrate an existing cluster to VPP
```

Comment:
Consider moving "Migrate an existing cluster to VPP" to /maintenance section.

Reply:
Moved, along with the IPsec page which I think also fits better there.